src/lib/y2network/routing_helpers.rb

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2network/interface"
+require "y2network/routing_table"
+require "y2network/route"
+
+module Y2Network
+  module RoutingHelpers
+
+  private
+
+    MISSING_VALUE = "-".freeze
+    private_constant :MISSING_VALUE
+    DEFAULT_DEST = "default".freeze
+    private_constant :DEFAULT_DEST
+
+    # Find configured network interfaces
+    #
+    # Configured interfaces has a configuration (ifcfg file) assigned.
+    #
+    # @return [Array<Interface>] Detected interfaces
+    # @see Yast::NetworkInterfaces.Read
+    def find_interfaces
```

Comment:
maybe have this method in `Y2Network::Interface`?

Reply:
Maybe, and this basically is configured interfaces. BTW, for autoyast It should use the config.interfaces as them should be them already imported  or read by Lan